docs/documentation/tutorial.md

```diff
@@ -312,6 +315,29 @@ hood to explore the game tree and find good moves. The default uses 1000 iterati
 move, which isn't sufficient to create a bullet-proof tic-tac-toe player. This can be
 configured to adjust the bot strength.
 
+If you want to call `step` from the React client, it is passed as one of the
+props to your `<Board>` component. Adapting our previous example, we could add
+a call to `step` each time the human player clicks on a cell:
+
+```js
+class TicTacToeBoard extends React.Component {
+  onClick(id) {
+    if (this.isActive(id)) {
+      this.props.moves.clickCell(id);
+      this.props.events.endTurn();
+      setTimeout(() => this.props.step(), 1000);
```

Comment:
Let's just document this in `Client.md` for now.  I would prefer to not document it prominently in the tutorial as I expect this to change pretty soon.

Reply:
👍 Done in 88f6e00b